controller/src/beerocks/master/db/node.h

```diff
@@ -212,10 +212,10 @@ class node {
         sMacAddr bssid;
         uint8_t channel_utilization;
         uint16_t number_of_stas_currently_associated;
-        uint8_t estimated_service_info_field_ac_be[3];
-        uint8_t estimated_service_info_field_ac_bk[3];
-        uint8_t estimated_service_info_field_ac_vo[3];
-        uint8_t estimated_service_info_field_ac_vi[3];
+        std::vector<uint8_t> estimated_service_info_fields;
```

Comment:
I'm not entirely happy with this. If we were to actually use this information for any purpose, it would make more sense to have something like this:

```
struct estimated_service_info {
    bool is_valid;
    uint8_t data_format;
    uint8_t ba_window_size;
    ...
};
estimated_service_info estimated_service_info_be;
estimated_service_info estimated_service_info_vo;
estimated_service_info estimated_service_info_bk;
estimated_service_info estimated_service_info_vi;
```

However, that is a *lot* more work for no real benefit (at this time), so OK.

That said, I think the original was better: 4 explicit lists. The booleans can be replaced with a comparison with `{0, 0, 0}`. To make that comparison easy, we can use `std::array<uint8_t, 3>` instead of a C array. But this option is also a nitpick so only change it if you really think it's better.